pkgs/development/libraries/audio/lv2/default.nix

```diff
@@ -1,45 +1,73 @@
 { stdenv
 , lib
 , fetchurl
+, asciidoc
+, codespell
+, doxygen
+, gtk2
+, libsamplerate
 , libsndfile
 , pkg-config
 , python3
-, wafHook
+, meson
+, ninja
 , pipewire
+, serd
+, sord
+, gitUpdater
 }:
 
 stdenv.mkDerivation rec {
   pname = "lv2";
-  version = "1.18.2";
+  version = "1.18.10";
 
-  outputs = [ "out" "dev" ];
+  outputs = [ "out" "dev" "doc" ];
 
   src = fetchurl {
-    url = "https://lv2plug.in/spec/${pname}-${version}.tar.bz2";
-    sha256 = "sha256-TokfvHRMBYVb6136gugisUkX3Wbpj4K4Iw29HHqy4F4=";
+    url = "https://lv2plug.in/spec/${pname}-${version}.tar.xz";
+    hash = "sha256-eMUbzyG1Tli7Yymsy7Ta4Dsu15tSD5oB5zS9neUwlT8=";
   };
 
   nativeBuildInputs = [
+    asciidoc
+    codespell
+    doxygen
+    meson
+    ninja
     pkg-config
-    wafHook
-    python3
+    (python3.withPackages (pkgs: with pkgs; [
+      lxml
+      markdown
+      pygments
+      rdflib
+    ]))
+    serd
+    sord
   ];
 
   buildInputs = [
+    gtk2
```

Comment:
We specifically removed gtk2 dependency in the past since it is unmaintained and not really needed https://github.com/NixOS/nixpkgs/commit/beefeab222118fe1c88afb6e8260eab71406fae7

Reply:
Yeah, I was lazy and just pulled in everything that default `meson` build wanted:

```
lv2> Run-time dependency gtk+-2.0 found: NO (tried pkgconfig)
lv2> plugins/eg-sampler.lv2/meson.build:17:0: ERROR: Dependency "gtk+-2.0" not found, tried pkgconfig
```

I'll spend some time trying to disable those depends.

Reply:
Maybe just disable plug-ins https://gitlab.com/lv2/lv2/-/issues/57#note_1096060029

Reply:
Sounds good! Disabled plugins (and while at it tests and docs as they pull in various python libraries that did not like to cross-compile).